doc: fix incorrect info about serial port availability

gvancuts opened the pull request:
Fix incorrect information about the presence (or not) of a serial
port on Kaby Lake NUC platforms.

Tracked-On: #5812
Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

dbkinder commented:
@gvancuts @fitchbe So close.  The CI doc build failed but didn't report what the actual failure was.  When you changed Yes to No (and No to Yes) in the table, the edge characters of the table weren't aligned with the rest of the rows so the whole table was rejected by Sphinx.  I added and deleted a space on those two rows to re-align the right table edge, and all is well.

dbkinder reviewed (approved):
I force pushed a change to fix the table edge alignment that was causing the CI doc build to fail.

gvancuts commented:
> 
> 
> @gvancuts @fitchbe So close. The CI doc build failed but didn't report what the actual failure was. When you changed Yes to No (and No to Yes) in the table, the edge characters of the table weren't aligned with the rest of the rows so the whole table was rejected by Sphinx. I added and deleted a space on those two rows to re-align the right table edge, and all is well.

Oops.... my bad :-( I should have ran `make doc` but the changes were so trivial, or so I thought, that I did not do it...

gvancuts commented:
> 
> 
> I force pushed a change to fix the table edge alignment that was causing the CI doc build to fail.

Thanks!!
